zproject/settings.py

```diff
@@ -158,7 +158,7 @@ def get_secret(key):
                     'REGISTER_LINK_DISABLED': False,
                     'LOGIN_LINK_DISABLED': False,
                     'ABOUT_LINK_DISABLED': False,
-                    'FIND_TEAM_LINK_DISABLED': True,
+                    'FIND_TEAM_LINK_DISABLED': False,
```

Comment:
We don't actually want this enabled by default in production, so I skipped this change.